pages/guides/how-to-self-host-hermes.mdx

```diff
@@ -0,0 +1,224 @@
+You can use the following node providers to get access to a Pythnet RPC:
+
+- [Triton](https://triton.one)
+- [P2P](https://p2p.org)
+- [Blockdaemon](https://blockdaemon.com/)
+
+## Self-hosting on Kubernetes
+
+This section explains how to self-host Hermes on Kubernetes. It assumes that you have a basic understanding of Kubernetes and Helm. If you are new to Kubernetes, you can use the
+following resources to get started:
+
+- [Kubernetes Basics](https://kubernetes.io/docs/tutorials/kubernetes-basics/)
+- [Helm Quickstart](https://helm.sh/docs/intro/quickstart/)
+
+### Requirements
+
+- A running Kubernetes cluster: You can use [microk8s](https://microk8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/) to run a single node cluster on production machine.
+  - The cluster nodes should have sufficient resources to run the Pythnet RPC and Hermes. Pythnet RPC requires at least 4 CPU cores and 4GB of RAM and Hermes requires at least 1
+    CPU core and 1GB of RAM.
+- [Helm](https://helm.sh/) installed on your machine.
+
+### Steps
+
+#### Add Pythnet Helm repository
+
+Run the following command to add the Pythnet Helm repository:
+
+```bash
+helm repo add pyth-network https://pyth-network.github.io/charts
+```
+
+#### Hermes
+
+Create a file named `hermes-values.yaml` and put the Pythnet RPC Http and WebSocket endpoints in it like so:
+
+```yaml
+hermes:
+  pythnetHttpEndpoint: "http(s)://fill-this-out/"
+  pythnetWsEndpoint: "ws(s)://fill-this-out/"
+```
+
+Then, run the following command to install Hermes with the release name of `hermes` in your cluster:
+
+```bash
+helm install hermes pyth-network/hermes --values hermes-values.yaml
+```
+
+You can customize the Hermes configuration by adjusting the default values in the [hermes/values.yaml](https://github.com/pyth-network/charts/blob/main/charts/hermes/values.yaml).
+
+Hermes helm chart creates a Service of type 'ClusterIP' that exposes the Hermes service only internally within the cluster. To expose it publicly you can either change this to a
+`NodePort` or `LoadBalancer` service or use an Ingress controller.
+
+##### Expose Hermes using NodePort service type
+
+If you are running a single node cluster, you can use a NodePort to expose the Hermes service to the outside world. To do so, add the following values to the `hermes-values.yaml` file:
+
+```yaml
+service:
+  type: NodePort
+```
+
+Then, run the following command to upgrade the Hermes installation:
+
+```bash
+helm upgrade hermes pyth-network/hermes --values hermes-values.yaml
+```
+
+You can find the NodePort by running the following command:
+
+```bash
+kubectl get svc
+```
+
+You can use the port number listed under the `PORT(S)` column to connect to Hermes from outside the cluster or configure a web server (like nginx) to proxy requests to Hermes.
+
+#### Expose Hermes using LoadBalancer service type
```

Comment:
I suggest removing this section entirely. If people need a LoadBalancer-type service (they probably don't), they will know how to set this up on their own.